app/_src/mesh/features/provenance-verification-images.md

```diff
@@ -0,0 +1,165 @@
+
+## Examples
+
+### Prerequisites
+
+For both examples, you need to:
+
+1. Ensure `cosign` / `slsa-verifier` is installed.
+
+2. Ensure `regctl` is installed.
+
+3. Collect the necessary image details.
+
+4. Parse the `<manifest_digest>` for the image using `regctl`.
+
+   ```sh
+   regctl manifest digest <image>:<tag>
+   ```
+
+5. Set the `COSIGN_REPOSITORY` environment variable:
+
+   ```sh
+   export COSIGN_REPOSITORY=kong/notary
+   ```
+
+{:.important .no-icon}
+> The GitHub owner is case-sensitive (`Kong/kong-mesh` vs `kong/kong-mesh`).
+
+### Minimal example
+
+#### Using Cosign
+
+Run the `cosign verify-attestation ...` command:
+
+```sh
+cosign verify-attestation \
+   <image>:<tag>@sha256:<manifest_digest> \
+   --type='slsaprovenance' \
+   --certificate-oidc-issuer='https://token.actions.githubusercontent.com' \
+   --certificate-identity-regexp='^https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v[0-9]+.[0-9]+.[0-9]+$'
+```
+
+Here's the same example using sample values instead of placeholders:
+
+```sh
+cosign verify-attestation \
+   'kong/kuma-cp:2.7.4@sha256:<manifest_digest>' \
+   --type='slsaprovenance' \
+   --certificate-oidc-issuer='https://token.actions.githubusercontent.com' \
+   --certificate-identity-regexp='^https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v[0-9]+.[0-9]+.[0-9]+$'
+```
+
+The command will exit with `0` when the `cosign` verification is completed:
+
+```sh
+...
+echo $?
+0
+```
+
+#### Using slsa-verifier
+
+Run the `slsa-verifier verify-image ...` command:
+
+```sh
+slsa-verifier verify-image \
+   <image>:<tag>@sha256:<manifest_digest> \
+   --print-provenance \
+   --source-uri 'github.com/Kong/<repo>'
+```
+
+Here's the same example using sample values instead of placeholders:
+
+```sh
+slsa-verifier verify-image \
+   'kong/kuma-cp:2.7.4@sha256:<manifest_digest>' \
+   --print-provenance \
+   --source-uri 'github.com/Kong/kong-mesh'
+```
+
+The command will print "Verified SLASA provenance" if successful:
+
+```sh
+...
+PASSED: Verified SLSA provenance
+```
+
+### Complete example
+
+#### Using Cosign
+
+Run the `cosign verify-attestation ...` command:
+
+```sh
+cosign verify-attestation \
+   <image>:<tag>@sha256:<manifest_digest> \
+   --type='slsaprovenance' \
+   --certificate-oidc-issuer='https://token.actions.githubusercontent.com' \
+   --certificate-identity-regexp='^https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v[0-9]+.[0-9]+.[0-9]+$' \
+   --certificate-github-workflow-repository='Kong/<repo>' \
+   --certificate-github-workflow-name='<workflow name>' \
+   --certificate-github-workflow-trigger='<workflow trigger>'
+```
+
+Here's the same example using sample values instead of placeholders:
+
+```sh
+cosign verify-attestation \
+   'kong/kuma-cp:2.7.4@sha256:<manifest_digest>' \
```

Comment:
replace `<manifest_digest>` with image digest for official 2.7.4 tag. Provenance is NOT generated for `preview` tags to test